plugin.video.retrospect/resources/menus/menu.py

```diff
@@ -0,0 +1,315 @@
+#===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+#===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+#===============================================================================
+
+import os
+import sys
+import xbmc
+import xbmcgui
+
+# we need to import the initializer
+addOnPath = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.append(addOnPath)
```

Comment:
why `sys.path.append`?

Reply:
Yes, the menu.py is used for the context menus. These are called as normal separate Python scripts and need access the main classes from the add-on.

Reply:
if they are in the path why not just use regular imports? sys.path.append is usually a bad practice